core/src/main/java/org/springframework/security/oauth2/server/authorization/OAuth2Authorization.java

```diff
@@ -21,11 +21,15 @@
 
 /**
  * @author Joe Grandja
+ * @author Madhu Bhat
  */
 public class OAuth2Authorization {
```

Comment:
Javadoc for class is required as per issue requirements.

Reply:
Added only to the setter that I added and not to the class as there is an open PR with javadoc for the class.